example/src/App.js

```diff
@@ -89,6 +88,80 @@ function App() {
     }
   }, [location]);
 
+  const oscalObjectTypes = [
+    {
+      pathName: "catalog",
+      elementName: "Catalog",
+      loaderElement: OSCALCatalogLoader,
+    },
+    {
+      pathName: "profile",
+      elementName: "Profile",
+      loaderElement: OSCALProfileLoader,
+    },
+    {
+      pathName: "component-definition",
+      elementName: "Component Definition",
+      loaderElement: OSCALComponentLoader,
+    },
+    {
+      pathName: "system-security-plan",
+      elementName: "System Security Plan",
+      loaderElement: OSCALSSPLoader,
+    },
+  ];
+
+  const appBarRoutes = (
+    <Route path="/">
+      <Route index element={`OSCAL Catalog ${appType}`} />
+      {oscalObjectTypes.map((oscalObjectType) => (
+        <Route path={oscalObjectType.pathName} key={oscalObjectType.pathName}>
+          {["", ":id"].map((path) => (
```

Comment:
Because we use `["", ":id"]` twice, can we extract it to a variable and then use that in each place?